types/data/options.d.ts

```diff
@@ -0,0 +1,15 @@
+export interface Options {

```

Comment:
Make use of this where applicable: https://github.com/hivesolutions/yonius/blob/2daa42b115756a3c22d346696d98cac647e0fc7c/types/data/model.d.ts#L3

Also, you might have to split this into `getOptions` and `findOptions`.

Given the scope of this, it should probably go inside `model.d.ts`.

Reply:
The `get` and `find` are equal, from what I can see in the code